crates/bevy_ui/src/lib.rs

```diff
@@ -36,6 +36,8 @@ mod render;
 mod stack;
 mod ui_node;
 
+pub mod actions;
```

Comment:
The "pub use module::*`" style below is pretty terrible for documentation: both because "pile of stuff" and because it breaks module docs.

I've broken from convention here: I would like to change the other modules in this crate in another PR.